packages/data/src/items/utils.js

```diff
@@ -77,8 +78,10 @@ export function searchItemsByString( select, endpoint, search ) {
 	search.forEach( ( searchWord ) => {
 		const query = {
 			search: searchWord,
-			per_page: 10,
+			...{ per_page: 10 },
```

Comment:
Sorry a nitpick, I'm not sure though why is this required to be spread? If i'm not mistaken, we could do simply like:
```suggestion
			per_page: 10,
```

Reply:
Thank you for the catch 👍 I think I was confused with something else when I worked on this 🤦 updated in https://github.com/woocommerce/woocommerce-admin/pull/7385/commits/61f230acb7385c3b7574dbaadc3b5e8281e1a0f0